src/link/persisted-queries/index.ts

```diff
@@ -93,18 +93,13 @@ function operationDefinesMutation(operation: Operation) {
     d => d.kind === 'OperationDefinition' && d.operation === 'mutation');
 }
 
-const { hasOwnProperty } = Object.prototype;
-
-const hashesByQuery = new WeakMap<
-  DocumentNode,
-  Record<string, Promise<string>>
->();
-
-let nextHashesChildKey = 0;
-
 export const createPersistedQueryLink = (
   options: PersistedQueryLink.Options,
 ) => {
+  const hashesByQuery = new WeakMap<
+  DocumentNode,
+  Promise<string>
+>();
```

Comment:
```suggestion
    DocumentNode,
    Promise<string>
  >();
```

[nit] Ensure indentation is correctly applied